redpanda/values.yaml

```diff
@@ -13,135 +13,201 @@
+  tag: v21.11.16
   # The imagePullPolicy will default to Always when the tag is 'latest'
-  tag: v21.11.15
-
-# Additional labels to apply to all Kubernetes resources created by this chart.
-labels: {}
-  # app.kubernetes.io/part-of: my-app
-
-clusterDomain: cluster.local
-
-# See https://vectorized.io/docs/configuration/
-config:
-  # If you have a license key please set it here or you can request one
-  # here: https://vectorized.io/redpanda
-  license_key: ""
-
-  # Update with your organization name
-  organization: ""
-
-  redpanda:
-    enable_sasl: false
-    admin:
-      # The port for the admin server.
-      #
-      # Metrics are served off of this port at /metrics.
-      port: 9644
-      address: 0.0.0.0
-      name: admin
-      external:
-        enabled: false
-
-    # Kafka API listeners
+  pullPolicy: IfNotPresent
+# Your license key (optional)
+license_key: ""
+# Your organization name (optional)
+organization: ""
+#
+# Authentication
+auth:
+  #
+  # SASL configuration
+  sasl:
+    enabled: false
+    # user list
+    users:
+      - admin
+  #
+  # TLS configuration
+  tls:
+    # Enable global TLS, which turns on TLS by default for all listeners
+    # Each listener must include a certificate name in its TLS section
+    # Any certificates in auth.tls.certs will still be loaded if enabled is false
+    # This is because listeners may enable TLS individually (see listeners.<listener name>.tls.enabled)
+    enabled: true
+    # list all certificates below, then reference a certificate's name in each listener (see listeners.<listener name>.tls.cert)
+    certs:
+      cert1:
+        caEnabled: true
+        # duration: 43800h
+        path: /etc/tls/certs/cert1
+#
+# Listeners
+listeners:
+  # Admin API listener
+  admin:
+    enabled: true
+    # The port for the admin server
+    # Metrics are served off of this port at /metrics
+    port: 9644
+    address: 0.0.0.0
+    external:
+      enabled: false
+    tls:
+      # Optional flag to override the global TLS enabled flag
+      # enabled: false
+      # Name of the certificate used for TLS. Enable TLS with auth.tls.enabled
+      # Must match a cert registered at auth.tls.certs
+      cert: cert1
+      # If true, the truststore file for this listener will be included in the ConfigMap
+      requireClientAuth: false
+  # Kafka API listeners
+  # The kafka listener group cannot be disabled
+  kafka:
     # The first entry is used to set up the headless service.
     # Kafka external and internal advertised listeners will be
     # derived from the configuration. External advertised listener will be via
     # node port unless load balancer is configured.
-    kafka_api:
-      - name: kafka
-        address: $(POD_IP)
-        port: 9092
-        external:
-          enabled: true
-          subdomain: "streaming.rockdata.io"
-
-    # rpc server listener
-    rpc_server:
-      address: 0.0.0.0
-      port: 33145
+    all:
```

Comment:
Each listener group has an all list that contains the associated listeners. This allows for having group-wide parameters such as `enabled`.

Reply:
I went back and forth on what to call this nested list. I didn't want to have listeners within listeners so just went with something short initially, but `endpoints` was on the short list of possibilities. Here's what it is now:
![image](https://user-images.githubusercontent.com/310946/173600595-c52ee46b-9cd7-4647-a650-72a5062c65a0.png)